_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP? Why is it so important to do a pre-GSoC analysis
+and design a proposal as early as possible?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC. The proposal is designed before the implementation of *major
+features* because the implementation usually happens from a technical
+perspective which may lead to a poorly designed content, both developer-wise
+and user-wise. So, to avoid this, a cEP should be designed at an earlier stage
+as this saves a lot of coding work and makes sure that the major features are
```

Comment:
`as this also saves` instead of `as saves` since multiple reasons are mentioned.